langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -15,6 +15,8 @@
 // under the License.
 
 import ballerina/jballerina.java;
+import ballerina/lang.'object;
```

Comment:
Import is not required.

Reply:
Without import it gives an `undefined module 'object'` error. Is this behavior incorrect?

Reply:
Could be the case that we don't define predeclared imports for langlib modules (which makes sense). So let's keep this as is.